pom.xml

```diff
@@ -2184,6 +2185,18 @@
         <artifactId>univocity-parsers</artifactId>
         <version>2.8.3</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.hive</groupId>
+        <artifactId>hive-storage-api</artifactId>
+        <version>2.6.0</version>
+        <scope>${hive.storage.scope}</scope>
+        <exclusions>
+          <exclusion>
+            <groupId>commons-lang</groupId>
```

Comment:
Is this needed for SBT?

Reply:
This is need for `Maven`, too.

Reply:
I'm rechecking this. I'll ping you again.

Reply:
Without this explicit management, `core` (not `sql/core`) fails like the following.
```
$ ./build/mvn -DskipTests --pl core --am package -Phadoop-3.2
...
[INFO] --- scala-maven-plugin:4.3.0:testCompile (scala-test-compile-first) @ spark-core_2.12 ---
[INFO] Using incremental compilation using Mixed compile order
[INFO] Compiler bridge file: /Users/dongjoon/.sbt/1.0/zinc/org.scala-sbt/org.scala-sbt-compiler-bridge_2.12-1.3.1-bin_2.12.10__52.0-1.3.1_20191012T045515.jar
[INFO] Compiling 271 Scala sources and 26 Java sources to /Users/dongjoon/PRS/SPARK-DEPENDENCY-CHANGE/core/target/scala-2.12/test-classes ...
[ERROR] [Error] /Users/dongjoon/PRS/SPARK-DEPENDENCY-CHANGE/core/src/test/scala/org/apache/spark/util/PropertiesCloneBenchmark.scala:23: object lang is not a member of package org.apache.commons
[ERROR] [Error] /Users/dongjoon/PRS/SPARK-DEPENDENCY-CHANGE/core/src/test/scala/org/apache/spark/util/PropertiesCloneBenchmark.scala:49: not found: value SerializationUtils
[ERROR] two errors found
```

Reply:
`test` dependency seems to be enough. I'll update this PR.